tests/integration/test_logcollector/test_options/test_restrict_regex.py

```diff
@@ -0,0 +1,269 @@
+
+components:
+    - logcollector
+
+suite: options
+
+targets:
+    - agent
+    - manager
+
+daemons:
+    - wazuh-logcollector
+
+os_platform:
+    - linux
+    - windows
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Windows 10
+    - Windows Server 2019
+    - Windows Server 2016
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/log-data-collection/index.html
+    - https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/localfile.html
+    - https://documentation.wazuh.com/current/user-manual/reference/statistics-files/wazuh-logcollector-state.html
+    - https://documentation.wazuh.com/current/user-manual/reference/internal-options.html#logcollector
+
+tags:
+    - logcollector_options
+'''
+import os
+import sys
+import re
+import pytest
+
+from wazuh_testing.tools import PREFIX
+from wazuh_testing.tools.local_actions import run_local_command_returning_output
+from wazuh_testing.tools.configuration import load_configuration_template, get_test_cases_data
+from wazuh_testing.modules.logcollector import event_monitor as evm
+from wazuh_testing.modules import logcollector as lc
+
+
+# Reference paths
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+
+# Configuration and cases data
+t1_configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_restrict_regex_default.yaml')
+t1_cases_path = os.path.join(TEST_CASES_PATH, 'cases_restrict_regex_default.yaml')
+
+t2_configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_restrict_regex_type_values.yaml')
+t2_cases_path = os.path.join(TEST_CASES_PATH, 'cases_restrict_regex_type_values.yaml')
+
+# Test configurations
+test_file = os.path.join(PREFIX, 'test')
+
+t1_configuration_parameters, t1_configuration_metadata, t1_case_ids = get_test_cases_data(t1_cases_path)
+for count, value in enumerate(t1_configuration_parameters):
+    t1_configuration_parameters[count]['LOCATION'] = test_file
+t1_configurations = load_configuration_template(t1_configurations_path, t1_configuration_parameters,
+                                                t1_configuration_metadata)
+
+t2_configuration_parameters, t2_configuration_metadata, t2_case_ids = get_test_cases_data(t2_cases_path)
+for count, value in enumerate(t2_configuration_parameters):
+    t2_configuration_parameters[count]['LOCATION'] = test_file
+t2_configurations = load_configuration_template(t2_configurations_path, t2_configuration_parameters,
+                                                t2_configuration_metadata)
+
+prefix = lc.LOG_COLLECTOR_PREFIX
+
+
+# Tests
+@pytest.mark.tier(level=1)
```

Comment:
```suggestion
@pytest.mark.tier(level=0)
```

Reply:
fixed in https://github.com/wazuh/wazuh-qa/pull/3582/commits/736cebbf8d76781caa2709ed9b8f305e31183c79